istioctl/cmd/add-to-mesh_test.go

```diff
@@ -143,6 +149,18 @@ func TestAddToMesh(t *testing.T) {
 				"https://istio.io/docs/setup/kubernetes/additional-setup/requirements/\n",
 			namespace: "default",
 		},
+		{
+			description: "valid case - add deployment into mesh",
+			args: strings.Split("experimental add-to-mesh deployment details-v1 --meshConfigFile testdata/mesh-config.yaml"+
+				" --injectConfigFile testdata/inject-config.yaml"+
+				" --valuesFile testdata/inject-values.yaml", " "),
+			expectedException: false,
+			k8sConfigs:        cannedK8sConfigs,
+			expectedOutput: "deployment details-v1.default updated successfully with Istio sidecar injected.\n" +
+				"Next Step: Add related labels to the deployment to align with Istio's requirement: " +
```

Comment:
`Add 'app' and 'version' labels for Istio telemetry:`

Reply:
This is actually coming from original add-to-mesh command.